server.js

```diff
@@ -40,11 +43,9 @@ client.on('message', msg => {
   }
   
   else if (msg.content.startsWith('imageSearchDaddy')) {
-    if (msg.author.id == '272120805016469518') {
-      msg.reply('Nice try')
-    } else if (msg.author.id == '357636864267780099') {
+    if (IMAGE_BANNED_IDS.indexOf(msg.author.id) !== -1) {
```

Comment:
can we use IMAGE_BANNED_IDS.includes(msg.author.id) instead? it look nicer hahaha good idea tho!

Reply:
Too used to browser based JS lol, sure!